src/components/Pagination/Pagination.tsx

```diff
@@ -1 +1,80 @@
-export const Pagination = () => {};
+import classNames from 'classnames';
+import React, { Dispatch, SetStateAction } from 'react';
+
+type Props = {
+  currentPage: number;
+  total: number;
+  postPerPage: number;
+  onPageChange:(pageNumber: number) => void;
+  setCurrentPage:Dispatch<SetStateAction<number>>;
+
+};
+
+export const Pagination: React.FC<Props>
+= ({
+  currentPage, total, postPerPage, onPageChange, setCurrentPage,
+}) => {
+  const pageNumbers = [];
+
+  for (let i = 1; i <= Math.ceil(total / postPerPage); i += 1) {
+    pageNumbers.push(i);
+  }
+
+  const previousPage = () => {
+    setCurrentPage((current: number) => current - 1);
+  };
+
+  const nextPage = () => {
+    setCurrentPage((current: number) => current + 1);
```

Comment:
if after fix `area-disabled` some tests still fail - check if it's not disabled before setting `currentPage`